source/content/caching-advanced-topics.md

```diff
@@ -52,6 +52,47 @@ For CSS or JavaScript changes, Drupal and WordPress each offer methods to ensure
 
 Dev and Multidev environments do not cache static assets.
 
+Another solution to consider when debugging issues related to stale static assets is to load the file via a custom PHP script that sets a `no-cache` header value, for example: 
+
+```php
+<?php
+
+// Punch through the Pantheon edge cache.
+header("Cache-Control: no-cache, must-revalidate");
+
+// Set JSON header
+header('Content-Type: application/json');
+
+// Echo file content
+echo file_get_contents("/path/to/file.json");
+```
+
+`pantheon_clear_edge_paths` is a platform function that can be implemented in a [Quicksilver hook (like `deploy`)](https://pantheon.io/docs/quicksilver#hooks) to automatically purge a given static asset from cache, for example: 
```

Comment:
Todo add usage such as 10 max limit, and wildcard paths not supported

Reply:
The pantheon_advanced_page_cache module uses this function from a Drupal module, so I don't think this is limited to just Quicksilver hooks.